config/routes.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  mount Rswag::Ui::Engine => '/api-docs'
```

Comment:
Since we're not going to expose the docs from the Rails server in production let's take these out. We're going to have them generated and hosted within the existing hugo site.